linux_admin.gemspec

```diff
@@ -26,17 +26,19 @@ registration, updates, etc.
   spec.executables   = `git ls-files -- bin/*`.split("\n")
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.0.0"
+  spec.required_ruby_version = Gem::Requirement.new(">= 2.6")
 
   spec.add_development_dependency "manageiq-style"
   spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec",    "~> 3.0"
   spec.add_development_dependency "rubocop"
 
   spec.add_dependency "awesome_spawn",        "~> 1.6"
+  spec.add_dependency "bcrypt_pbkdf",         ">= 1.0", "< 2.0"
+  spec.add_dependency "ed25519",              ">= 1.2", "< 1.3"
   spec.add_dependency "inifile"
   spec.add_dependency "more_core_extensions", "~> 4.0"
-  spec.add_dependency "net-ssh", "~> 4.2.0"
+  spec.add_dependency "net-ssh",              "~> 7.2.3"
```

Comment:
@bdunne Please review - do we know what the follow on effect of this is?
@agrare I'm wondering if we should release this as a major? (at a minimum a minor)

Reply:
Since we need to drop a supported ruby versions yes this needs to be at least a minor bump

Reply:
Breaking changes since 4.2.0:
`Breaking change: ed25519 now requires ed25519 gem instead of RbNaCl gem [#563]` - https://github.com/net-ssh/net-ssh/blob/master/CHANGES.txt#L127
`BREAKING: Drop support for Ruby 2.5` - https://github.com/net-ssh/net-ssh/blob/master/CHANGES.txt#L37